Nodejs/Product/TestAdapter/TestFrameworks/ExportRunner/exportrunner.js

```diff
@@ -85,6 +85,7 @@ var run_tests = function (testCases, callback) {
             result.title = test.testName;
             testCase[test.testName]();
             result.passed = true;
+            result.stdOut += "Test passed.\n";
```

Comment:
Is this what appears in the Test Explorer 'output' link? Shouldn't you do the same for when the test fails in the 'catch' clause then (maybe with the exception message in the output too)?